aws_batch/pipeline.py

```diff
@@ -886,28 +886,30 @@ def run_gsnapl_remotely(sample, input_fa_1, input_fa_2,
     execute_command("aws s3 cp %s %s/" % (gsnap_ssh_key_s3_path, REF_DIR))
     key_path = REF_DIR +'/' + key_name
     execute_command("chmod 400 %s" % key_path)
-    commands =  "mkdir -p /home/ec2-user/batch-pipeline-workdir/%s;" % sample
-    commands += "aws s3 cp %s/%s /home/ec2-user/batch-pipeline-workdir/%s/ ; " % \
-                 (sample_s3_output_path, input_fa_1, sample)
-    commands += "aws s3 cp %s/%s /home/ec2-user/batch-pipeline-workdir/%s/ ; " % \
-                 (sample_s3_output_path, input_fa_2, sample)
-    commands += " ".join(['/home/ec2-user/bin/gsnapl',
+    remote_home_dir = "/home/ubuntu"
```

Comment:
why don't you just do 
remote_work_dir = ""/home/ubuntu/batch-pipeline-workdir/%s" % sample  ?

Reply:
need it in several different paths, see below

Reply:
need it in several different paths, see below

Reply:
can probably do both 
remote_home_dir 
remote_work_dir 
to make the code more succint though.